os/board/common/partitions.c

```diff
@@ -25,6 +25,8 @@
 #include <errno.h>
 #include <string.h>
 #include <sys/types.h>
+#include <stdlib.h>
+#include <stdio.h>
```

Comment:
Could you add this next to config.h?
```
#include <tinyara/config.h>

#include <stdio.h>
#include <stdlib.h>
#include <debug.h>
...
```

Reply:
fixed

Reply:
I don't know what you fixed. I mean I want to position them next of config.h as shown above. Anyway I merge this now.